evcouplings/compare/protocol.py

```diff
@@ -60,7 +60,7 @@ def _filter_by_id(x, id_list):
             "by_alignment", "alignment_min_overlap",
             "sequence_id", "sequence_file", "region",
             "use_bitscores", "domain_threshold",
-            "sequence_threshold", "jackhmmer",
+            "sequence_threshold", "hmmer",
```

Comment:
remove checking for "jackhmmer" altogether here (and also no "hmmer"); as we are branching later, the parameters will be verified in the particular function/protocol call.